windows.md

```diff
@@ -397,7 +397,59 @@ With those compatibility things out of the way, you're ready to start the system
     ```
 
     It should print out a welcome message like this:<br><br>
-    <img src="macos-6-docker-hello-world.png"><br><br>
+    <img src="macos-7-docker-hello-world.png"><br><br>
+
+    If opening Docker Desktop or running the command above shows a message that says `Docker Desktop - Windows HyperVisor is not Present`, you may need to enable virtualization on your machine.
+
+    The error might look like this:<br><br>
+    <img src="docker-desktop-opening-error.avif"><br><br>
+
+    First check if virtualization is enabled on your computer: To do this, open PowerShell as an administrator and run the following command:
+    ```bash
+    systeminfo | findstr /C:"Hyper-V" /C:"Virtualization"
+    ```
+
+    This should display the output below: Note the `Virtualization Enabled In Firmware: No`, this shows that virtualization is not enabled on your computer
+
+    ```bash
+    Hyper-V Requirements: VM Monitor Mode Extensions: Yes
+                          Virtualization Enabled In Firmware: No
+    ```
+
+    Continue with the steps below if virtualization is not enabled, Enable virtualization in your BIOS or UEFI:
+
+    1. Start your computer if it is off, or restart it if it has already started
```

Comment:
spend a few edit cycles to try to reduce the length of the whole text

one example of possible simplification of this line: instead of saying "either do this or that", and building a concept of state into it, just give one instruction that is probably going to be the case for 90% of students (in this case "Restart your computer")

Reply:
Overall instructions changed

e9dcf99d92e43034c3071d66ca2dd15ccfe271b0